build/package/mac_notarize.sh

```diff
@@ -22,17 +22,36 @@ set -Eeou pipefail
 
 # this script could run in parallel for both x86_64 and arm64
 # we need to make sure to call the right one at the right time
+
+curl "${NOTARY_SERVICE_URL}" --output macos-notary.zip
+unzip -u macos-notary.zip
+chmod 755 ./darwin_amd64/macnotary
```

Comment:
can we make this a separate step as install gon was?

Reply:
updated